src/TesApi.Web/Runner/TaskToNodeTaskConverter.cs

```diff
@@ -424,15 +424,43 @@ private TesInput PrepareLocalFileInput(TesInput input, string defaultStorageAcco
             });
         }
 
-        private static void MapInputs(List<TesInput> inputs, string pathParentDirectory, string containerMountParentDirectory,
+        private async Task MapInputsAsync(List<TesInput> inputs, string pathParentDirectory, string containerMountParentDirectory,
             NodeTaskBuilder builder)
         {
-            inputs?.ForEach(input =>
+            if (inputs == null || inputs.Count == 0)
```

Comment:
`inputs == null` --> `inputs is null`

Consider: `if ((inputs?.Count ?? 0) == 0)`